partiql-spi/src/main/java/org/partiql/eval/value/Datum.java

```diff
@@ -534,17 +534,19 @@ static Datum missing(@NotNull PType type) {
     }
 
     @NotNull
-    static Datum bag(@NotNull Iterable<Datum> values) {
-        return new DatumCollection(values, PType.typeBag());
+    static Datum bool(boolean value) {
+        return new DatumBoolean(value);
     }
 
+    // NUMERIC
+
     @NotNull
-    static Datum tinyInt(byte value) {
-        return new DatumByte(value, PType.typeTinyInt());
+    static Datum tinyint(byte value) {
```

Comment:
Any reason you opted to remove the camel-case? Also, does this leave a green-squiggly on your IDE?

Reply:
No squiggly, but the naming is quite simple – spaces are underscores in snake, or capital for camel.

However, note that most types don't have spaces.

**Examples**

```
TINYINT -- sql
tinyint() -- method

DOUBLE PRECISION -- sql
doublePrecision() -- method

VARCHAR -- sql
varchar() -- method

CHARACTER VARYING -- sql
characterVarying() -- method
```

Not everything is that consistent because of reserved words in Java, and I've chosen to shorten some redundant things – for example `TIME_WITH_TIMEZONE` is easy to shorten to [TIMEZ](https://duckdb.org/docs/sql/data_types/time.html)